pwndbg/heap/ptmalloc.py

```diff
@@ -94,10 +94,7 @@ def contains_chunk(self, size, chunk):
 
             # TODO: Refactor this, the bin should know how to calculate
             # largebin_index without calling into the allocator
```

Comment:
just to be sure: is this comment still relevant?

Reply:
Yes, it's still relevant. With an ideal design, there should be no need to call `pwndbg.heap.current.largebin_index(size)`, as the algorithm for doing that doesn't depend on having a reference to an actual allocator. Not a big deal right now, but this separation will help in the long run.